ci/tools/lookup-run-id

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# A utility script to find the GitHub Actions workflow run ID for a given git tag.
+# This script looks for the CI workflow run that corresponds to the commit of the given tag.
+
+set -euo pipefail
+
+# Check required arguments
+if [[ $# -lt 2 ]]; then
+    echo "Usage: $0 <git-tag> <repository> [workflow-name]" >&2
```

Comment:
What are the `>&2` bits intended to do here?

Reply:
Ah, standard error instead of the default standard out. Not sure that is needed but far from critical if you want it.

Reply:
I thought it's typical that the usage msg is printed to stderr?

Reply:
I don't see it so much, but it probably doesn't hurt. For usage I thought it was standard out, for errors probably standard error is more normal. There are several below that look like they belong in standard out and the AI just defaulted everything to error for some reason.